tests/RuleSet/DeclarationBlockTest.php

```diff
@@ -505,4 +505,50 @@ public function canRemoveCommentsFromRulesUsingStrictParsing(
 
         self::assertSame($cssWithoutComments, $renderedDocument);
     }
+
+    /**
+     * Asserts two declaration blocks are equivalent, allowing the rules to be in any order.
+     *
+     * @param string $expected
+     * @param string $actual
+     */
+    public static function assertDeclarationBlockEquals($expected, $actual)
+    {
+        $normalizedExpected = self::sortRulesInDeclarationBlock($expected);
+        $normalizedActual = self::sortRulesInDeclarationBlock($actual);
+
+        self::assertSame($normalizedExpected, $normalizedActual);
+    }
+
+    /**
+     * Sorts the rules within a declaration block by property name.
+     *
+     * @param string $declarationBlock
+     *
+     * @return string
+     */
+    private static function sortRulesInDeclarationBlock($declarationBlock)
+    {
+        // Match everything between `{` and `}`.
+        return \preg_replace_callback(
+            '/(?<=\{)[^\}]*+(?=\})/',
```

Comment:
Let's escape the backslashes to make it clear to the reader that they aren't intended to escape them for the PHP string, but for the regular expression.

```suggestion
            '/(?<=\\{)[^\\}]*+(?=\\})/',
```

Reply:
The backslashes are escaping the `{` and `}`.

Reply:
Ah, yes, I see.